ramp4-pcar4/ramp4-pcar4
# Improve tailwind breakpoints handling for teleported panels

For #1729, #1780.

### Changes
- Modified the resize observer to take in whatever breakpoints you please.
- Made teleported panels' breakpoints configurable with the defaulting according to [this logic](https://github.com/ramp4-pcar4/ramp4-pcar4/issues/1729#issuecomment-1589946581).
- Fixed a minor bug where the console would yell when you wanted a grid config but no merge grids in it.
- Updated docs.

### Testing
1. Open the grid on the [teleport WET](https://ramp4-pcar4.github.io/ramp4-pcar4/1729/demos/index-teleport-wet) sample.
2. Notice the custom breakpoints for the grid defined [here](https://github.com/mohsin-r/ramp4-pcar4/blob/1729/demos/starter-scripts/teleport-wet.js#L457).
3. Open the browser's dev tools and make your window responsive. Try resizing the window and check the grid's panel container (the direct child of the teleport target) in the elements section to see if the correct classes are present or not.


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/ramp4-pcar4/ramp4-pcar4/1781)
